### ISSUE TITLE
Moved to https://github.com/Previews-app/Previews/issues

### ISSUE THREAD

**Issue by MarkM-dev** [MarkM-dev/Previews#1]

**Comment by frank479** [comment]
So, I'm having the same issue I reported back in December with my Favorites list being corrupted.  I tried to use the link to your explanation of how to fix this, but the link is no good and it appears the responses were moved.  But, I still can't find them.  It used to be here:  https://github.com/MarkM-dev/Previews/issues/106#issuecomment-1321251243
I remember it having something to do with exporting the preferences but where to go from there... I'm lost.  Can  you help?

**Comment by MarkM-dev** [comment]
if i remember correctly, you had favorites that you needed to remove manually.
you should be able to remove them now via the settings menu edit button next to the feature.

if not, then 
- export the favorites
- delete the object with the name you want to remove, for example, an object with mediumstein would look like this:
{"display_name":"mediumstein","profile_pic_url":"","stream_name":"mediumstein"}
Dont forget to also remove a comma (,).
- go to the extension settings and click import and select the file you just saved,

**Comment by frank479** [comment]
Yes!  That was it.  And, yes, I see the feature you added to remove those.  Thank you so much!
